### PR TITLE
Edit, Delete Student

### PR DESCRIPTION
**Pull request by rvente**

### CODE REVIEW

**Comment by rvente** [comment]
Hold for review until  Web-Dev-Speedrunners/flex-or-fail/pull/29 is merged so we can validate delete

**Comment by rvente** [comment]
Works.

**Review by xuejingao — APPROVED** [review]

File: src/store/thunks.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import * as ac from "./actions/actionCreators";
 const axios = require("axios");
+const BACKEND_URL =  "https://r1chy5zeeh.execute-api.us-east-1.amazonaws.com"
```
Comment (shikdernyc) [inline src/store/thunks.js:3]: Can you add this in a different file, like `config.js`?

**Review by shikdernyc — APPROVED** [review]
Optional change requested

**Pull request merged**